docs/conf.py

```diff
@@ -158,8 +158,8 @@
 
 
 def setup(app):
-    app.add_stylesheet('custom.css')
-    app.add_stylesheet('highlight.css')
```

Comment:
when trying to preview the `.rst` files in VSCode it was failing with this error: https://github.com/sphinx-doc/sphinx/issues/7747. The change I made fixes it for me, is it ok to make? I'm not too familiar with Sphinx usage